### ISSUE TITLE
Version mismatch

### ISSUE THREAD

**Issue by HelloWorldLTY** [scverse/pertpy#678]
### Report

Hi, I also found that the pertpy from pip and pertpy from github are in different version. For example, milo plot function is not included in the pip install version. Would you please condsider revising it? Thanks.

### Version information

absl-py                   1.4.0
adjustText                0.7.3
aiohttp                   3.8.5
aiosignal                 1.3.1
anndata                   0.9.2
annotated-types           0.5.0
anyio                     4.0.0
argon2-cffi               23.1.0
argon2-cffi-bindings      21.2.0
arrow                     1.2.3
arviz                     0.15.1
asciitree                 0.3.3
asttokens                 2.2.1
async-lru                 2.0.4
async-timeout             4.0.3
attrs                     23.1.0
Babel                     2.12.1
backcall                  0.2.0
backports.zoneinfo        0.2.1
beautifulsoup4            4.12.2
bleach                    6.0.0
cached-property           1.5.2
cachetools                5.3.1
certifi                   2023.7.22
cffi                      1.15.1
chardet                   5.2.0
charset-normalizer        3.2.0
chex                      0.1.7
click                     8.1.7
cloudpickle               2.2.1
cmake                     3.27.2
comm                      0.1.4
contextlib2               21.6.0
contourpy                 1.1.0
custom-inherit            2.4.1
cycler                    0.11.0
dask                      2023.5.0
dask-image                2023.3.0
debugpy                   1.6.7.post1
decorator                 5.1.1
decoupler                 1.6.0
defusedxml                0.7.1
dm-tree                   0.1.8
docrep                    0.3.2
entrypoints               0.4
et-xmlfile                1.1.0
etils                     1.3.0
exceptiongroup            1.1.3
executing                 1.2.0
fasteners                 0.18
fastjsonschema            2.18.0
filelock                  3.12.3
flax                      0.7.2
fonttools                 4.42.1
fqdn                      1.5.1
frozenlist                1.4.0
fsspec                    2023.6.0
future                    0.18.3
google-auth               2.22.0
google-auth-oauthlib      1.0.0
grpcio                    1.57.0
h5netcdf                  1.1.0
h5py                      3.9.0
idna                      3.4
igraph                    0.10.6
imageio                   2.31.2
importlib-metadata        6.8.0
importlib-resources       6.0.1
inflect                   7.0.0
ipykernel                 6.25.1
ipython                   8.12.2
ipython-genutils          0.2.0
ipywidgets                8.1.0
isoduration               20.11.0
jax                       0.4.13
jaxlib                    0.4.13
jaxopt                    0.8.3
jedi                      0.19.0
Jinja2                    3.1.2
joblib                    1.3.2
json5                     0.9.14
jsonpointer               2.4
jsonschema                4.19.0
jsonschema-specifications 2023.7.1
jupyter                   1.0.0
jupyter_client            8.3.1
jupyter-console           6.6.3
jupyter_core              5.3.1
jupyter-events            0.7.0
jupyter-lsp               2.2.0
jupyter_server            2.7.3
jupyter_server_terminals  0.4.4
jupyterlab                4.0.5
jupyterlab-pygments       0.2.2
jupyterlab_server         2.24.0
jupyterlab-widgets        3.0.8
kiwisolver                1.4.5
lazy_loader               0.3
leidenalg                 0.9.1
lightning-utilities       0.9.0
lit                       16.0.6
llvmlite                  0.39.1
locket                    1.0.0
Markdown                  3.4.4
markdown-it-py            3.0.0
MarkupSafe                2.1.3
matplotlib                3.7.2
matplotlib-inline         0.1.6
matplotlib-scalebar       0.8.1
mdurl                     0.1.2
mistune                   3.0.1
mizani                    0.9.3
ml-collections            0.1.1
ml-dtypes                 0.2.0
mpmath                    1.3.0
msgpack                   1.0.5
mudata                    0.2.3
multidict                 6.0.4
multipledispatch          1.0.0
muon                      0.1.6
natsort                   8.4.0
nbclient                  0.8.0
nbconvert                 7.8.0
nbformat                  5.9.2
nest-asyncio              1.5.7
networkx                  3.1
notebook                  7.0.3
notebook_shim             0.2.3
numba                     0.56.4
numcodecs                 0.11.0
numpy                     1.22.4
numpyro                   0.11.0
nvidia-cublas-cu11        11.10.3.66
nvidia-cuda-cupti-cu11    11.7.101
nvidia-cuda-nvrtc-cu11    11.7.99
nvidia-cuda-runtime-cu11  11.7.99
nvidia-cudnn-cu11         8.5.0.96
nvidia-cufft-cu11         10.9.0.58
nvidia-curand-cu11        10.2.10.91
nvidia-cusolver-cu11      11.4.0.1
nvidia-cusparse-cu11      11.7.4.91
nvidia-nccl-cu11          2.14.3
nvidia-nvtx-cu11          11.7.91
oauthlib                  3.2.2
omnipath                  1.0.7
openpyxl                  3.1.2
opt-einsum                3.3.0
optax                     0.1.7
orbax-checkpoint          0.2.3
ott-jax                   0.4.6
overrides                 7.4.0
packaging                 23.1
pandas                    1.5.3
pandocfilters             1.5.0
parso                     0.8.3
partd                     1.4.0
patsy                     0.5.3
pertpy                    0.4.0
pertvi                    0.0.1
pexpect                   4.8.0
pickleshare               0.7.5
Pillow                    10.0.0
PIMS                      0.6.1
pip                       23.2.1
pkgutil_resolve_name      1.3.10
platformdirs              3.10.0
plotnine                  0.10.1
ply                       3.11
prometheus-client         0.17.1
prompt-toolkit            3.0.36
protobuf                  3.20.1
psutil                    5.9.5
ptyprocess                0.7.0
pure-eval                 0.2.2
py-pcha                   0.1.3
pyasn1                    0.5.0
pyasn1-modules            0.3.0
pycparser                 2.21
pydantic                  2.3.0
pydantic_core             2.6.3
pyDeprecate               0.3.1
Pygments                  2.16.1
pynndescent               0.5.10
Pyomo                     6.8.0
pyparsing                 3.0.9
pypi-latest               0.1.2
pypng                     0.20220715.0
PyQt5                     5.15.11
PyQt5-Qt5                 5.15.15
PyQt5_sip                 12.15.0
pyro-api                  0.1.2
pyro-ppl                  1.8.6
python-dateutil           2.8.2
python-json-logger        2.0.7
pytorch-lightning         1.7.7
pytz                      2023.3
PyWavelets                1.4.1
PyYAML                    6.0.1
pyzmq                     25.1.1
qtconsole                 5.4.3
QtPy                      2.4.0
questionary               2.0.1
referencing               0.30.2
reportlab                 4.2.5
requests                  2.31.0
requests-oauthlib         1.3.1
rfc3339-validator         0.1.4
rfc3986-validator         0.1.1
rich                      13.5.2
rpds-py                   0.10.0
rpy2                      3.5.16
rsa                       4.9
scanpy                    1.9.4
scikit-image              0.21.0
scikit-learn              1.3.0
scikit-misc               0.1.4
scipy                     1.10.1
scvi-tools                0.19.0
seaborn                   0.11.2
Send2Trash                1.8.2
session-info              1.0.0
setuptools                75.2.0
simvi                     0.1.0
six                       1.16.0
sklearn                   0.0.post7
slicerator                1.1.0
sniffio                   1.3.0
soupsieve                 2.4.1
sparsecca                 0.3.1
squidpy                   1.2.3
stack-data                0.6.2
statannotations           0.5.0
statsmodels               0.14.0
stdlib-list               0.9.0
switchlang                0.1.0
sympy                     1.12
tensorboard               2.14.0
tensorboard-data-server   0.7.1
tensorstore               0.1.41
terminado                 0.17.1
texttable                 1.6.7
threadpoolctl             3.2.0
tifffile                  2023.7.10
tinycss2                  1.2.1
tomli                     2.0.1
toolz                     0.12.0
torch                     2.0.1
torch-geometric           2.3.1
torchmetrics              0.11.4
tornado                   6.3.3
toyplot                   2.0.0
toytree                   2.0.5
tqdm                      4.66.1
traitlets                 5.9.0
triton                    2.0.0
typing_extensions         4.7.1
tzdata                    2023.3
tzlocal                   5.2
umap-learn                0.5.3
uri-template              1.3.0
urllib3                   1.26.16
validators                0.21.2
wcwidth                   0.2.6
webcolors                 1.13
webencodings              0.5.1
websocket-client          1.6.2
Werkzeug                  2.3.7
wheel                     0.41.2
widgetsnbextension        4.0.8
wrapt                     1.15.0
xarray                    2023.1.0
xarray-einstats           0.5.1
yarl                      1.9.2
zarr                      2.16.1
zipp                      3.16.2

**Comment by Zethson** [comment]
They should be very similar. Please fill out the issue template! Which versions are you using?

**Comment by HelloWorldLTY** [comment]
Hi I have updated, running pip install pertpy -U gives me 0.4.0.

**Comment by Zethson** [comment]
Yeah this is ancient. The latest version on pypi is https://pypi.org/project/pertpy/ 0.9.4 . You must be using an old Python version. Please use 3.10 or 3.11 in a virtual environment.
